src/tracking/envelope.cpp

```diff
@@ -123,6 +124,13 @@ namespace impactx
                                        << " slice_step=" << slice_step << "\n";
                     }
 
+                    if (space_charge)
+                    {
+                        // push Covariance Matrix in 2D space charge fields
+                        amrex::ParticleReal current=0.5;  //TODO: This must be set.
```

Comment:
This is hard-coded to the benchmark example value of 0.5 A, until the ParmParse logic is added.